src/person/controllers/update_person_controller.py

```diff
@@ -20,23 +21,30 @@ def process(self):
         net_id = self._data.get("net_id")
         first_name = self._data.get("first_name")
         last_name = self._data.get("last_name")
+        major_ids = self._data.get("majors")
         hometown = self._data.get("hometown")
         profile_pic_base64 = self._data.get("profile_pic_base64")
         facebook_url = self._data.get("facebook_url")
         instagram_username = self._data.get("instagram_username")
         graduation_year = self._data.get("graduation_year")
         pronouns = self._data.get("pronouns")
-
         goals = self._data.get("goals")
         talking_points = self._data.get("talking_points")
-
         availability = self._data.get("availability")
         locations = self._data.get("locations")
         groups = self._data.get("groups")
         interests = self._data.get("interests")
-
         has_onboarded = self._data.get("has_onboarded")
 
+        if major_ids is not None:
```

Comment:
`major_ids` here but below you had them named `locations` and `groups` instead of `location_ids` and `group_ids` - choose a naming convention and stick to it